### PR TITLE
Fix the storage section to be easier to understand

### LINKED ISSUE #124: Fix UG section 6 to be easier to read
Apparently its an absolute disaster and it's complete gobbledegook. We need to fix this. (Correction: I made the mess so I need to fix it)

### PR DESCRIPTION
**Pull request by NeilBaner**
Fixes #124 hopefully.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1083,38 +1088,108 @@ Read Book
...
+
+#### 6.2.4. The Notebook Shelf
+
+The notebook shelf is what is stored in the save file. It represents all your notebooks, sections, and pages at once. It is formatted as follows. 
+
+The first line contains just a number. This number indicates the number of notebooks in your Notebook Shelf. 
+
+Then, all the notebooks on your Notebook Shelf are reprsesented as described in [6.2.3. Notebooks](#6.2.3.-Notebooks). 
+
+For example, a saved notebook shelf might look like this:
```
Comment (franceneee) [inline docs/UserGuide.md]: This section may seem a bit confusing? I think putting a screenshot here would be better.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1083,38 +1088,108 @@ Read Book
...
 
-The first line in the `j`th set contains the title of the `j`th section of the `i`th notebook.
+```
+Java
+2
+What is Java?
+Java is a programming language used by 3 billion devices.
+Why Java?
+It's highly portable and relatively fast.
+```
```
Comment (franceneee) [inline docs/UserGuide.md]: Less technical now, yay.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1083,38 +1088,108 @@ Read Book
...
+
+For example, a page might be represented like this: 
+
+```
+What is Java?
+Java is a programming language used by 3 billion devices.
+```
+
+The first line, that says "What is Java", is the title of the page, while the second line is the content of that page. 
+
```
Comment (franceneee) [inline docs/UserGuide.md]: Better than before. :)

Comment (NeilBaner) [reply docs/UserGuide.md]: phew thank you

File: docs/UserGuide.md (hunk 1 above)
Comment (NeilBaner) [reply docs/UserGuide.md]: I'll see what I can come up with.

**Review by franceneee — APPROVED** [review]

**Pull request merged**